app/data/contactAPI.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * The ContactAPI is a JavaScript object that contains an array to hold the contact objects,
+ * and methods for manipulating the contacts array or speciifc objects within the array
+ **/
+
+export const ContactAPI = {
+  contacts: [],
+  all: function () {
```

Comment:
don't be afraid of being even more clear
```suggestion
  allContacts: function () {
```

Reply:
Why not es6?
use arrow functions!